src/components/DuplicatesPage/DuplicatesList/DuplicatesList.js

```diff
@@ -0,0 +1,35 @@
+import DuplicatesTable from "../DuplicatesTable/DuplicatesTable";
+import * as React from "react";
+import PropTypes from "prop-types";
+import {areDuplicatesMerged} from "../../../utils/duplicates";
+import MergedItem from "../MergedItem/MergedItem";
+import "./duplicatesList.less";
+
+const renderDuplicatesTable = (duplicates, actions) => (
+    <DuplicatesTable duplicates={duplicates} actions={actions}/>
+);
+
+const renderMergedItem = (duplicates) => (
+    <MergedItem duplicate={duplicates.data[0]}/>
+);
+
+const getDuplicateTitle = (duplicate) => (duplicate.firstName + ' ' + duplicate.lastName).toUpperCase();
+
```

Comment:
methods should be inside Component Function (render methods). GetDuplicateTitle ---> transformTitleToUppercase? This name doesn't have any semantic right now. It might be managed by CSS too.